SciCatProject/catanie
# Upgrade/angular 10.2

## Description

Upgrade Angular from v9 to v10

## Motivation

Upgrade

## Changes:

* Upgrade Angular from v9 to v10
* Add ngcc postinstall script
* Add ng lint posttest script
* Remove deprecated async import in favour of waitForAsync in spec files
* Remove `entryComponents` from modules, as it's no longer needed
* Remove deprecated `TestBed.get` in favour of `TestBed.inject`
* Add stricter checks to `angularCompilerOptions`
*  Remove comment on `node_modules` in .dockerignore
* Remove greenkeeper config from package.json, as greenkeeper no longer exists
* Set default style extension to `scss`

## Tests included/Docs Updated?

- [ ] Included for each change/fix?
- [x] Passing? (Merge will not be approved unless this is checked) 
- [x] New packages used/requires npm install? 
- [ ] Toggle added for new features?
- [ ] Requires update of catamel API?
